jujugui/static/gui/src/app/components/inspector/inspector.js

```diff
@@ -152,7 +152,7 @@ class Inspector extends React.Component {
               destroyUnits={nextProps.destroyUnits}
               previousComponent={previousComponent}
               service={service}
-              showSSHButtons={nextProps.showSSHButtons}
+              showSSHButtons={nextProps.showSSHButtons && window.juju_config.flags.expert}
```

Comment:
I think we normally try and avoid accessing these globals inside the components, but if this is temporary I guess it should be OK.